packages/platform/src/lib/nx-plugin/generators/app/generator.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  addDependenciesToPackageJson,
+  formatFiles,
+  generateFiles,
+  getWorkspaceLayout,
+  names,
+  offsetFromRoot,
+  stripIndents,
+  Tree,
+} from '@nrwl/devkit';
+import * as path from 'path';
+import { AnalogNxApplicationGeneratorOptions } from './schema';
+import { lt, major } from 'semver';
+import { getInstalledAngularVersion } from '../../utils/version-utils';
+import { runTasksInSerial } from '@nrwl/workspace/src/utilities/run-tasks-in-serial';
+import { addAnalogProjectConfig } from './lib/add-analog-project-config';
+
+export interface NormalizedOptions extends AnalogNxApplicationGeneratorOptions {
+  projectName: string;
+  projectRoot: string;
+  projectDirectory: string;
+  parsedTags: string[];
+}
+
+function normalizeOptions(
+  tree: Tree,
+  options: AnalogNxApplicationGeneratorOptions
+): NormalizedOptions {
+  const name = names(options.name).fileName;
```

Comment:
suggestion: we already called `names()` here, let's return the value we need from `names()` as part of `NormalizeOptions` as well so we don't have to rerun `names()` in `addFiles()`